src/test/java/org/kohsuke/github/GHIssueEventTest.java

```diff
@@ -41,11 +43,11 @@ public void testEventsForSingleIssue() throws Exception {
     public void testRepositoryEvents() throws Exception {
         GHRepository repo = getRepository();
         List<GHIssueEvent> list = repo.listIssueEvents().toList();
-        assertTrue(list.size() > 0);
+        assertThat(list.size() > 0, is(true));
```

Comment:
this is a more normal way of doing it
```suggestion
        assertThat(list, is(not(empty()));
```

Reply:
Yeah, there's a bunch of these kind of changes. I was thinking I could get by without them, but good call. I'll go through them once and pick off any easy ones like this.